File: terceiroSemestre/poo/polimorfismo/java/src/ContatoJuridico.java
public class ContatoJuridico() extends Contato{
    private String cnpj;
    private String razaoSocial;

    ContatoJuridico(String nome, String endereço, String email, 
        String cnpj, String razaoSocial){
        super(nome, endereço, email);
        this.cnpj = cnpj;
        this.razaoSocial = razaoSocial;
    }

    String toString(){
        super.toString();
        System.out.println("CNPJ: " + cnpj);
        System.out.println("Razão Social: " + razaoSocial);
    }
}

File: terceiroSemestre/poo/polimorfismo/python/ex1/contato_juridico.py
import contato

class ContatoJuridico(contato.Contato):
    def __init__(self, nome, endereço, email, razaoSocial, cnpj):
        super().__init__(nome, endereço, email)
        self.razaoSocial = razaoSocial
        self.cnpj = cnpj

    def __str__(self):
        super().__str__()
        print("Razão Social: " + self.razaoSocial)
        print("CNPJ = " + self.cnpj)

    